### PR TITLE
Update bmvc paper code

### PR DESCRIPTION
**Pull request by lscheinkman**
Copied changes from https://github.com/numenta/nupic.research/pull/592

@nielsleadholm Please review

### CODE REVIEW

**Comment by lscheinkman** [comment]
@nielsleadholm any update of the URL/DOI for the paper ?

**Comment by nielsleadholm** [comment]
> @nielsleadholm any update of the URL/DOI for the paper ?

No unfortunately still waiting, will let you know as soon as I here more. The official conference start is November 22nd, so may be a couple more weeks.

**Comment by nielsleadholm** [comment]
> @nielsleadholm any update of the URL/DOI for the paper ?

No unfortunately still waiting, will let you know as soon as I here more. The official conference start is November 22nd, so may be a couple more weeks.

**Comment by nielsleadholm** [comment]
@lscheinkman 

Hi Luiz, the link is available now https://www.bmvc2021-virtualconference.com/conference/papers/paper_1506.html

File: bmvc/grid_cell_path_integration_for_movement_based_visual_object_recognition/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Purpose of the code
 ===============
-The following implements a network using grid-cell computations that can perform basic object recognition on visual tasks such as the MNIST data-set. It builds on the architecture described in [Lewis et al 2019](https://www.frontiersin.org/articles/10.3389/fncir.2019.00022/full), enabling similar principles to be applied to image-based tasks. 
+The following implements a network using grid-cell computations that can perform basic object recognition on visual tasks such as the MNIST data-set. It builds on the architecture described in [Lewis et al 2019](https://www.frontiersin.org/articles/10.3389/fncir.2019.00022/full), enabling similar principles to be applied to image-based tasks. The repository can be used to generate the results found in our BMVC 2021 paper, "Grid Cell Path Integration For Movement-Based Visual Object Recognition".
```
Comment (subutai) [inline bmvc/grid_cell_path_integration_for_movement_based_visual_object_recognition/README.md:3]: Maybe link to the paper in this README as well? We could just add a URL to the title.

**Review by subutai — APPROVED** [review]

**Review by nielsleadholm — APPROVED** [review]

**Pull request merged**